pkg/rpm_pfg.bzl

```diff
@@ -45,7 +45,7 @@ spec_filetype = [".spec", ".spec.in", ".spec.tpl"]
 # TODO(nacl, #292): cp -r does not do the right thing with TreeArtifacts
 _INSTALL_FILE_STANZA_FMT = """
 install -d "%{{buildroot}}/$(dirname '{1}')"
-cp '{0}' '%{{buildroot}}/{1}'
+cp '%{{_topdir}}/BUILD/{0}' '%{{buildroot}}/{1}'
```

Comment:
Could we add a comment explaining where `_topdir` comes from and what it means in this context? I presume that [this](https://github.com/bazelbuild/rules_pkg/blob/3d32d9f694647955623dbf75173bb2b0e4afeea7/pkg/make_rpm.py#L358) is the source for `_topdir`. Is that correct?

Reply:
@cgrindel Sorry for the noise, was just updating my branch from main before I started hacking on it this morning.   After further debugging I believe that I have found the root cause for the issue (which I had initially assumed was due to using an older rpmbuild version). 

see: https://github.com/bazelbuild/rules_pkg/pull/792 for the more correct solution